include/abb_librws/rws_interface.h

```diff
@@ -131,11 +143,18 @@ class RWSInterface
      *
      * \param name for the name of the task.
      * \param is_motion_task indicating if the task is a motion task or not.
+     * \param is_active indicating if the task is active or not.
+     * \param execution_state indicating the task's current execution state.
      */
-    RAPIDTaskInfo(const std::string& name, bool is_motion_task)
+    RAPIDTaskInfo(const std::string& name,
+                  const bool is_motion_task,
+                  const bool is_active,
+                  const RAPIDTaskExecutionState execution_state)
```

Comment:
Is this a breaking change (ie: users could already be using this)?

Reply:
No, I think that is very unlikely.

The construction of `RAPIDTaskInfo` objects should only occur inside the `getRAPIDTasks(...)` method [here](https://github.com/jontje/abb_librws/blob/1f8478e5ff115129f0454208212263d33f709ada/src/rws_interface.cpp#L288).